features/gebruik_in_lopende_tekst.feature

```diff
@@ -30,6 +31,7 @@ Rule: gebruikInLopendeTekst voor een persoon zonder adellijke titel of predicaat
     | N                     | GA VV GN-VP GP        |
   - De eerste naamcomponent direct na "mevrouw" of "de heer" begint met een hoofdletter
   - Wanneer een naamcomponent geen of een lege waarde heeft, wordt de overbodige spatie niet opgenomen: niet starten met een spatie, niet eindigen met een spatie, geen dubbele spatie, geen spatie na streepje
+  - Aanduiding naamgebruik "E" (eigen naam) wordt gehanteerd voor een persoon die geen actuele en geen ontbonden huwelijken/partnerschappen heeft gehad
```

Comment:
Heb je dit toegevoegd om een fout-situatie te corrigeren ? Of mag iemand die geen actuele en geen ontbonden huwelijke/partnerschappen heeft ook aanduiding naamgebruik  "P" krijgen volgens het LO ?

Reply:
dit heb ik overgenomen uit feature aanschrijfwijze. Het gaat hier eigenlijk om inconsistente data in de bron. Deze (sub)regel beschrijft hoe daarmee moet worden omgegaan. Dit mag overigens wel volgens LO